source/Cosmos.Common/BinaryHelper.cs

```diff
@@ -1,133 +1,138 @@
 ﻿using System;
 
-namespace Cosmos.Common
+namespace Cosmos.Common;
```

Comment:
I dont think we should change the namespace format everywhere.

Reply:
Its the Auto-formatter from JetBrains Rider